satpy/etc/readers/seviri_l2_grib.yaml

```diff
@@ -73,244 +73,274 @@ datasets:
     name: error_in_upper_layer_cloud_optical_depth
+    long_name: Upper Cloud Optical Depth Error Estimate
+    standard_name: atmosphere_optical_thickness_due_to_cloud standard_error
     resolution: 3000.403165817
     file_type: grib_seviri_oca
     parameter_number: 34
     units: "1"
-    long_name: cloud_optical_depth
 
   error_in_upper_layer_cloud_top_pressure:
     name: error_in_upper_layer_cloud_top_pressure
+    long_name: Upper Cloud Top Pressure Error Estimate
+    standard_name: air_pressure_at_cloud_top standard_error
     resolution: 3000.403165817
     file_type: grib_seviri_oca
     parameter_number: 35
     units: Pa
-    standard_name: air_pressure_at_cloud_top_standard_error
 
   error_in_upper_layer_cloud_effective_radius:
     name: error_in_upper_layer_cloud_effective_radius
+    long_name: Upper Cloud Particule Effective Radius Error Estimate
+    standard_name: effective_radius_of_cloud_condensed_water_particles_at_cloud_top standard_error
     resolution: 3000.403165817
     file_type: grib_seviri_oca
     parameter_number: 36
     units: m
-    standard_name: effective_radius_of_cloud_condensed_water_particles_at_cloud_top_standard_error
 
   lower_layer_cloud_optical_depth:
     name: lower_layer_cloud_optical_depth
+    long_name: Lower Cloud Optical Depth
+    standard_name: atmosphere_optical_thickness_due_to_cloud_in_lower_atmosphere_layer
     resolution: 3000.403165817
     file_type: grib_seviri_oca
     parameter_number: 37
     units: "1"
-    long_name: cloud_optical_depth
 
   lower_layer_cloud_top_pressure:
     name: lower_layer_cloud_top_pressure
+    long_name: Lower Cloud Top Pressure
+    standard_name: air_pressure_at_cloud_top_in_lower_atmosphere_layer
     resolution: 3000.403165817
     file_type: grib_seviri_oca
     parameter_number: 38
     units: Pa
-    standard_name: air_pressure_at_cloud_top
 
   error_in_lower_layer_cloud_optical_depth:
     name: error_in_lower_layer_cloud_optical_depth
+    long_name: Lower Cloud Optical Depth Error Estimate
+    standard_name: atmosphere_optical_thickness_due_to_cloud_in_lower_atmosphere_layer standard_error
     resolution: 3000.403165817
     file_type: grib_seviri_oca
     parameter_number: 39
     units: "1"
-    long_name: cloud_optical_depth_standard_error
 
   error_in_lower_layer_cloud_top_pressure:
     name: error_in_lower_layer_cloud_top_pressure
+    long_name: Lower Cloud Top Pressure Error Estimate
+    standard_name: air_pressure_at_cloud_top_in_lower_atmosphere_layer standard_error
     resolution: 3000.403165817
     file_type: grib_seviri_oca
     parameter_number: 40
     units: Pa
-    standard_name: air_pressure_at_cloud_top_standard_error
 
   fire_probability:
     name: fire_probability
+    long_name: Fire Probability
+    standard_name: fire_probability
     resolution: 3000.403165817
     file_type: grib_seviri_fir
     parameter_number: 192
     units: "%"
-    long_name: fire_probability
 
   active_fires:
```

Comment:
Please add `flag_values` and `flag_meanings`.

Reply:
done